src/test/javascript/spec/component/markdown-editor/ordered-list-command.spec.ts

```diff
@@ -43,6 +43,60 @@ describe('OrderedListCommand', () => {
     it('should remove ordered list on execute', () => {
         comp.aceEditorContainer.getEditor().setValue('1. line 1\n2. line 2');
         command.execute();
-        expect(comp.aceEditorContainer.getEditor().getValue()).toBe('line 1\nline 2\n');
+        expect(comp.aceEditorContainer.getEditor().getValue()).toBe('line 1\nline 2');
+    });
+
+    it('should handle empty lines and create new list', () => {
+        comp.aceEditorContainer.getEditor().setValue('Test\n\nTest');
+        command.execute();
+        expect(comp.aceEditorContainer.getEditor().getValue()).toBe('1. Test\n\n2. Test');
+    });
+
+    it('should handle empty lines and remove lists', () => {
+        comp.aceEditorContainer.getEditor().setValue('1. Test\n2.\n3. Test');
+        command.execute();
+        expect(comp.aceEditorContainer.getEditor().getValue()).toBe('Test\n\nTest');
+    });
+
+    it('should handle empty lines and remove lists', () => {
```

Comment:
```suggestion
    it('should handle multiple sequential empty lines and remove lists', () => {
```
The test above has the same name. I suppose Jest doesn’t care but that potentially might create problems when some test report XML parser wants to display the results.

Reply:
Yes you are right. That could be a problem :+1: